internal/credential/vault/repository.go

```diff
@@ -6,6 +6,9 @@ import (
 	"github.com/hashicorp/boundary/internal/kms"
 )
 
+// OidcRepoFactory is used by "service functions" to create a new oidc repo
+type CredentialRepoFactory func() (*Repository, error)
```

Comment:
Copy/paste error for the comment.
"Credential" prefix may be redundant, consider removing.